IMA-WorldHealth/bhima
# Added asset registry action menu item to show scans for the asset

Add an action menu item for the Assets Registry that will jump to the Assets Scan Registry and only show the scans for that asset.

Note that this PR removed a couple of unused translation items in the asset.json files.

Closes https://github.com/IMA-WorldHealth/bhima/issues/6482

**TESTING**
- Use bhima_test
- Go to Asset Management > Asset Registry
- In the action menu for an asset, select the "Show Asset Scans"
- This should jump to the "Asset Scans Registry" page but filtered down to the specified asset

- Check the french translation for ASSET.SHOW_ASSET_SCANS
